cmd/collector/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM centos:7
```

Comment:
I thought we'd put all Docker files in one dir. But no strong feeling either way.